### PR TITLE
Added HMS integration to dev app

### PR DESCRIPTION
**Pull request by jkasten2**
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-android-sdk/1078)

### CODE REVIEW

File: OneSignalSDK/app/build.gradle [hunk 1]
```diff
@@ -68,6 +68,8 @@ dependencies {
 
     // For Chrome tabs
     implementation 'com.android.support:customtabs:28.0.0'
+
+    implementation 'com.huawei.hms:push:4.0.3.301'
```
Comment (mikechoch) [inline OneSignalSDK/app/build.gradle:72]: Should we also add in the location dependency?
`implementation 'com.huawei.hms:location:X.X.X.X'`

File: OneSignalSDK/app/src/main/java/com/onesignal/example/HmsMessageServiceAppLevel.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     * This method is called in the following cases:
+     *   1. "Data messages" - App process is alive when received.
+     *   2. "Notification Message" - foreground_show = false and app is in focus
+     * This method callback must be completed in 10 seconds. Start a new Job if more time is needed.
+     *
+     * @param message RemoteMessage
+     */
+    @Override
+    public void onMessageReceived(RemoteMessage message) {
+        Log.e("HMS", "onMessageReceived: " + message);
```
Comment (Jeasmine) [inline OneSignalSDK/app/src/main/java/com/onesignal/example/HmsMessageServiceAppLevel.java:35]: is ok to be error lvl log?

**Comment by jkasten2** [comment]
Going to wait until after the major release to merge, since the dev is app has changed in the 4.0.0 branch.

**Review by emawby — APPROVED** [review]
Approving with new changes

**Pull request merged**